Swarm fails to find packages when running unit testing with `go test -race ./...`!

Hi, All,

When running unit tests locally on my own machine, it always reports error like these:
```
root@ubuntu:~/gocode/src/github.com/docker/swarm# go test -race ./...
vendor/github.com/Sirupsen/logrus/examples/hook/hook.go:5:2: cannot find package "gopkg.in/gemnasium/logrus-airbrake-hook.v2" in any of:
	/root/gocode/src/github.com/docker/swarm/vendor/gopkg.in/gemnasium/logrus-airbrake-hook.v2 (vendor tree)
	/usr/local/go/src/gopkg.in/gemnasium/logrus-airbrake-hook.v2 (from $GOROOT)
	/root/gocode/src/gopkg.in/gemnasium/logrus-airbrake-hook.v2 (from $GOPATH)
```

missing packages in vendor, GOROOT and GOPATH.

Maybe we need to add it in the /vendor, but now /vendor dir misses vendor.json, thus still broken.
https://github.com/docker/swarm/pull/2220 has tried to make vendor complete.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -155,7 +155,7 @@ For complete documentation on how to use Swarm, refer to the Swarm section of [d
 To run unit tests:
 
 ```sh
-go test -race ./...
+go test -v -race `go list ./... | grep -v /vendor/`
 ```
 
 To run integration tests:

